src/sales-channels/asc/pricing-products.md

```diff
@@ -21,27 +21,25 @@ You can add pricing rules for your Amazon listings. Price rules can be used to a
 
-Using intelligent repricing to adjust your products listing prices on Amazon, based on competitor's pricing. Amazon Sales Channel has built in safeguards for you to set to protect margins or avoid matching the prices of a merchant with low feedback. Using intelligent repricing rules, Amazon listing prices can be automatically manipulated as a fixed or percentage amount (up or down) or even synchronized to the Buy Box or Lowest Price on a per item basis. Rules can even be stacked to provide unlimited flexibility.
-
-Pricing rules determine if your product is listed on Amazon using your defined Price Source or if the price is automatically adjusted before listing on Amazon, based on defined conditions for the rule.
+Using intelligent repricing to adjust your Amazon listing prices, based on competitor's pricing. Amazon Sales Channel has built in safeguards for you to configure to protect margins or avoid matching the prices of a merchant with low feedback. Using [intelligent repricing rules]({% link sales-channels/asc/intelligent-repricing-rules.md %}), Amazon listing prices can be automatically manipulated as a fixed or percentage amount (up or down) or even synchronized to the [Buy Box]({% link sales-channels/asc/buy-box-competitor-pricing.md %}) price or [Lowest Competitor Price]({% link sales-channels/asc/lowest-competitor-pricing.md %}) on a per item basis. Rules can even be stacked to provide unlimited flexibility.
 
 You can control important aspects of rules, such as active/inactive status, website eligibility, optional date ranges, and optional priority levels (used for rule stacking).
 
 For example, you can define and set the conditions for a price rule that, when the conditions are met, automatically adjust your listing price before it is sent to Amazon.
 
-Another pricing option is a price override, which is set at the individual listing level. A price override can be set, and an override ignores/takes priority over all other defaults, settings, and rules. An override can be set for price, handling time, condition, and seller notes (with a few exceptions). See [Overrides]({% link sales-channels/asc/overrides.md %}).
+Another pricing option is a [price override]({% link sales-channels/asc/overrides.md %}), which is set at the individual listing level. A [price override]({% link sales-channels/asc/overrides.md %}) can be set, and an override ignores/takes priority over all other defaults, settings, and rules. An [override]({% link sales-channels/asc/overrides.md %}) can be set for price, handling time, condition, and seller notes (with a few exceptions).
```

Comment:
two links to the same place in a single paragraph?

Reply:
yes.  since they were so close, i thought it looked odd to link one and not the other.